src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,51 @@
+.calendar {
+  width: calc(7 * 100px + 6px);
+  height: 300px;
+  @include flex;
+  margin: 0 auto;
+  padding: 50px 10px;
+
+  &__day {
+    width: $box-size;
+    height: $box-size;
+    @include font();
+    background-color: $bg-color;
+    border: 1px solid $color-black;
+    transition: 0.5s;
+    box-sizing: border-box;
+    border-radius: 3px;
```

Comment:
#12 in checklist - **Remember about styles properties order - ([css order](https://codeguide.academy/html-css.html#css-order))**